backend/src/main/java/pt/ua/deti/ies/SmartHomes/backend/WebSocket/SocketIOConfig.java

```diff
@@ -0,0 +1,57 @@
+package pt.ua.deti.ies.SmartHomes.backend.WebSocket;
+
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+
+import com.corundumstudio.socketio.Configuration;
+import com.corundumstudio.socketio.SocketIOClient;
+import com.corundumstudio.socketio.SocketIOServer;
+import com.corundumstudio.socketio.listener.ConnectListener;
+import com.corundumstudio.socketio.listener.DisconnectListener;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.CrossOrigin;
+
+
+@CrossOrigin
+@Component
+@Log4j2
+public class SocketIOConfig {
+
+	@Value("${socket.host}")
```

Comment:
These values must be defined in the app properties from compose.yaml (both dev + prod).
```yaml
environment:
      SPRING_APPLICATION_JSON: '{
          "spring.datasource.url" : "jdbc:postgresql://db_pg:5432/smarthomes",
          "spring.datasource.username" : "postgres",
          "spring.datasource.password" : "postgres",
          "spring.jpa.hibernate.ddl-auto" : "update",
          "spring.influx.url" : "http://influx:8086",
          "spring.influx.user" : "influx",
          "spring.influx.password" : "influx_smarthomes",
          "spring.influx.database" : "smarthomes",
          "spring.influx.token" : "smarthomes",
          "spring.influx.bucket" : "smarthomes",
          "spring.influx.org" : "smarthomes",
          "spring.rabbitmq.host" : "rabbitmq",
          "spring.rabbitmq.port" : "5672",
          "spring.rabbitmq.username" : "rabbitmq",
          "spring.rabbitmq.password" : "rabbitmq",
          "spring.rabbitmq.queue.json.name" : "smarthomes",
          "spring.rabbitmq.routing.json.key" : "smarthomes_routing_json_key",
          "spring.socket.host" : [bla bla bla],
          "spring.socket.port" : [bla bla bla],
        }'
```